src/controller/webrtc/WebRTCClient.cpp

```diff
@@ -144,48 +113,71 @@ CHIP_ERROR WebRTCClient::CreatePeerConnection(const std::string & stunUrl)
         return CHIP_ERROR_POSIX(errno);
     }
 
+    return CHIP_NO_ERROR;
+}
+
+void WebRTCClient::addVideoTrack(std::string mid, int payloadType)
+{
+    if (mVideoTrack != nullptr)
+    {
+        ChipLogProgress(Camera, "Video track already added");
+        return;
+    }
+
+    std::string vMid = mid.empty() ? kVideoMid : mid;
```

Comment:
Will this use of mid as received address https://github.com/project-chip/connectedhomeip/issues/41125 or is that a separate problem?